lib/index.js

```diff
@@ -32,20 +32,19 @@ const BALENA_SDK_HAS_SET_SHARED_OPTIONS = 'BALENA_SDK_HAS_SET_SHARED_OPTIONS';
  */
 
 /**
- * @alias module:balena-sdk
  * @summary Creates a new SDK instance using the default or the provided options.
  *
  * @description
  * The module exports a single factory function.
  *
  * @example
- * const getSdk = require('balena-sdk');
- * const balena = getSdk({
+ * import { BalenaSdk } from 'balena-sdk';
+ * const balena = BalenaSdk({
  * 	apiUrl: "https://api.balena-cloud.com/",
  * 	dataDirectory: "/opt/local/balena"
  * });
  */
-const getSdk = function (opts) {
+export const BalenaSdk = function (opts) {
```

Comment:
I'd rename this to `fromOptions`, ie
```ts
import * as BalenaSdk from 'balena-sdk';
const sdk = BalenaSdk.fromOptions({...})
```
I used `fromOptions` instead of `get` to match the `fromSharedOptions` pattern